spec/features/user_can_create_a_conversation_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+feature 'User can create a new message' do
+    let(:user) { FactoryBot.create(:user, email:'user1_email@gmail.com', password:'password1' )}
+    before do 
+        visit root_path 
+        click_on 'Login'
+        fill_in 'Email', with: user.email
+        fill_in 'Password', with: user.password
+        click_on 'Log in'
+        click_on 'Inbox'
+        click_on 'Compose'
+    end 
+
+    describe 'user can view the compose new message fields' do 
```

Comment:
if they can send off a message then they can compose it

Reply:
The 'Compose' button from the welcome page seems to bring the user to the page containing the fields for the new message (or conversation). The next logical step would be to test the 'Send Message' button to complete the action but we weren't sure how to write this test.